src/modules/components/transaction/transactionDataListItem/transactionDataListItemStructure/transactionDataListItemStructure.tsx

```diff
@@ -0,0 +1,116 @@
+import classNames from 'classnames';
+import { useChains } from 'wagmi';
+import {
+    AvatarIcon,
+    DataList,
+    Heading,
+    IconType,
+    NumberFormat,
+    Spinner,
+    formatterUtils,
+    type AvatarIconVariant,
+} from '../../../../../core';
+import {
+    TransactionType,
+    TxStatusCode,
+    txHeadingStringList,
+    txIconTypeList,
+    txVariantList,
+    type ITransactionDataListItemProps,
+} from './transactionDataListItemStructure.api';
+
+export const TransactionDataListItemStructure: React.FC<ITransactionDataListItemProps> = (props) => {
+    const {
+        chainId,
+        tokenAddress,
+        tokenSymbol,
+        tokenValue,
+        fiatEstimate,
+        txType = TransactionType.ACTION,
+        txStatus = TxStatusCode.PENDING,
```

Comment:
These are set as required on the prop interface, if we want to set default values we should set them as optional and specify the default values on the interface